### ISSUE TITLE
Camera "Look" input 

### ISSUE THREAD

**Issue by AriRubenstein** [dreamworksanimation/dwa_usd_plugins#7]
It appears the new USD camera has lost its "Look" input functionality

**Comment by jegstad-dwa** [comment]
Correct, that logic was unfinished in the current code, it will be operational in the next release.

**Comment by jegstad-dwa** [comment]
@AriRubenstein 

The look connection on Axis/Camera nodes should be working now in the latest code update. However it's not 100% matching the Foundry implementation as the look-strength does not smoothly interpolate without scaling artifacts, so I'll need to address this.

Hopefully you don't use the strength control very often.

-jonathan
